core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -133,6 +127,11 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
       preparedSelectHighestSequenceNr
       preparedSelectDeletedTo
       preparedInsertDeletedTo
+      preparedSelectTagProgress
+      preparedSelectTagProgressForPersistenceId
+      preparedWriteToTagProgress
+      preparedWriteToTagView
+    case msg => log.warning("Bug: {}", msg)
```

Comment:
not sure this is better than the default `unhandled`

I'd be careful with toString of unknown things.

Reply:
lol didn't mean to commit this